app/components/views/DexPage/ConfirmDexSeed/ConfirmDexSeed.jsx

```diff
@@ -0,0 +1,73 @@
+import { useDex } from "../hooks";
+import { useState } from "react";
+import { Tooltip, classNames } from "pi-ui";
+import { KeyBlueButton, CopyToClipboardButton } from "buttons";
+import { FormattedMessage as T } from "react-intl";
+import styles from "./ConfirmDexSeed.module.css";
+
+const ConfirmDexSeed = () => {
+  const { onConfirmDexSeed, dexSeed } = useDex();
+  const [showSeed, setShowSeed] = useState(false);
+
+  const onToggleSeed = () => {
+    setShowSeed(!showSeed);
+  };
+
+  return (
+    <div>
+      <div
+        className={classNames(
+          styles.actions,
+          styles.isRow,
+          styles.seedInstructions
+        )}>
+        <T
+          id="dex.instructions.seed"
+          m="You should carefully write down your application
+      seed and save a copy. Should you lose access to this machine or the
+      critical application files, the seed can be used to restore your DEX accounts
+       and native wallets. Some older accounts cannot be restored from seed,
+       and whether old or new, it's good practice to backup the account keys
+       separately from the seed."
```

Comment:
I suggest we replace the sentence

> Some older accounts cannot be restored from seed, and whether old or new, it's good practice to backup the account keys separately from the seed

with 

> DEX accounts created in prior versions are not recoverable with this seed, so be sure to export any such accounts from the DEX Settings page.

If it were known here if we got to this page **from init**, we could just omit the sentence.  If we got here **from login**, then this could be an upgrade from dex 0.2 so the message would be of interest.

Reply:
Ref: https://github.com/decred/dcrdex/blob/7cb08b8a5d0a6864ccc18bbe05b2b1afec0b68d7/docs/release-notes/release-notes-0.4.0.md#upgrading